# Translations template for senlin.
# Copyright (C) 2016 ORGANIZATION
# This file is distributed under the same license as the senlin project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2016.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: senlin 0.2.0.dev570\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2016-01-17 22:40-0500\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.1.1\n"

#: senlin/api/common/serializers.py:62
#, python-format
msgid ""
"JSON body size (%(len)s bytes) exceeds maximum allowed size (%(limit)s "
"bytes)."
msgstr ""

#: senlin/api/common/wsgi.py:55
msgid ""
"Address to bind the server. Useful when selecting a particular network "
"interface."
msgstr ""

#: senlin/api/common/wsgi.py:58
msgid "The port on which the server will listen."
msgstr ""

#: senlin/api/common/wsgi.py:60
msgid "Number of backlog requests to configure the socket with."
msgstr ""

#: senlin/api/common/wsgi.py:63
msgid "Location of the SSL certificate file to use for SSL mode."
msgstr ""

#: senlin/api/common/wsgi.py:66
msgid "Location of the SSL key file to use for enabling SSL mode."
msgstr ""

#: senlin/api/common/wsgi.py:69
msgid "Number of workers for Senlin service."
msgstr ""

#: senlin/api/common/wsgi.py:71
msgid ""
"Maximum line size of message headers to be accepted. max_header_line may "
"need to be increased when using large tokens (typically those generated "
"by the Keystone v3 API with big service catalogs)."
msgstr ""

#: senlin/api/common/wsgi.py:76
msgid ""
"The value for the socket option TCP_KEEPIDLE.  This is the time in "
"seconds that the connection must be idle before TCP starts sending "
"keepalive probes."
msgstr ""

#: senlin/api/common/wsgi.py:86
msgid "If false, closes the client socket explicitly."
msgstr ""

#: senlin/api/common/wsgi.py:88
msgid ""
"Timeout for client connections' socket operations. If an incoming "
"connection is idle for this number of seconds it will be closed. A value "
"of '0' indicates waiting forever."
msgstr ""

#: senlin/api/common/wsgi.py:100
msgid ""
"Address for invoking webhooks. It is useful for cases where proxies are "
"used for triggering webhooks. Default to the hostname of the API node."
msgstr ""

#: senlin/api/common/wsgi.py:104
msgid ""
"The port on which a webhook will be invoked. Useful when service is "
"running behind a proxy."
msgstr ""

#: senlin/api/common/wsgi.py:112
msgid ""
"Maximum raw byte size of JSON request body. Should be larger than "
"max_template_size."
msgstr ""

#: senlin/api/common/wsgi.py:150
msgid ""
"When running server in SSL mode, you must specify both a cert_file and "
"key_file option value in your configuration file"
msgstr ""

#: senlin/api/common/wsgi.py:166
#, python-format
msgid "Could not bind to %(bind_addr)s after trying  30 seconds"
msgstr ""

#: senlin/api/common/wsgi.py:684
msgid ""
"The server could not comply with the request since it is either malformed"
" or otherwise incorrect."
msgstr ""

#: senlin/api/middleware/webhook.py:94
#, python-format
msgid "Webhook failed authentication: %s."
msgstr ""

#: senlin/api/openstack/v1/actions.py:33
msgid "No action name specified"
msgstr ""

#: senlin/api/openstack/v1/actions.py:38
msgid "No target specified"
msgstr ""

#: senlin/api/openstack/v1/actions.py:43
#: senlin/api/openstack/v1/clusters.py:301
msgid "No action specified"
msgstr ""

#: senlin/api/openstack/v1/clusters.py:68
msgid "No cluster name specified."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:71
msgid "No cluster desired capacity provided."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:75
msgid "No cluster profile provided."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:78
#: senlin/api/openstack/v1/clusters.py:94
msgid ""
"Cluster min_size, if specified, must be less than or equal to its desired"
" capacity."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:84
#: senlin/api/openstack/v1/clusters.py:100
msgid ""
"Cluster max_size, if specified, must be greater than or equal to its "
"desired capacity. Setting max_size to -1 means no upper limit on cluster "
"size."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:109 senlin/engine/service.py:458
msgid ""
"Cluster max_size, if specified, must be greater than or equal to its "
"min_size. Setting max_size to -1 means no upper limit on cluster size."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:175
#: senlin/api/openstack/v1/clusters.py:205
msgid "Malformed request data, missing 'cluster' key in request body."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:234
msgid "Missing number value for resize operation."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:239
msgid "Missing adjustment_type value for resize operation."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:252
#, python-format
msgid ""
"The specified min_size (%(n)s) is greater than the specified max_size "
"(%(m)s)."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:279
msgid "The data provided is not a map."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:283
msgid "The 'policy_id' field is missing in the request."
msgstr ""

#: senlin/api/openstack/v1/clusters.py:304
msgid "Multiple actions specified"
msgstr ""

#: senlin/api/openstack/v1/clusters.py:308
#, python-format
msgid "Unrecognized action '%s' specified"
msgstr ""

#: senlin/api/openstack/v1/clusters.py:314
msgid "No node to add"
msgstr ""

#: senlin/api/openstack/v1/clusters.py:320
msgid "No node to delete"
msgstr ""

#: senlin/api/openstack/v1/clusters.py:343
msgid "No policy specified for detach."
msgstr ""

#: senlin/api/openstack/v1/nodes.py:38
msgid "No node name specified."
msgstr ""

#: senlin/api/openstack/v1/nodes.py:46
msgid "No profile ID provided."
msgstr ""

#: senlin/api/openstack/v1/nodes.py:106 senlin/api/openstack/v1/nodes.py:136
msgid "Malformed request data, missing 'node' key in request body."
msgstr ""

#: senlin/api/openstack/v1/nodes.py:140
msgid ""
"Updating cluster_id is not allowed, please invoke cluster add/remove node"
" actions if needed."
msgstr ""

#: senlin/api/openstack/v1/policies.py:76
#: senlin/api/openstack/v1/policies.py:98
msgid "Malformed request data, missing 'policy' key in request body."
msgstr ""

#: senlin/api/openstack/v1/policies.py:80
msgid "No policy name specified"
msgstr ""

#: senlin/api/openstack/v1/policies.py:83
msgid "No policy spec provided"
msgstr ""

#: senlin/api/openstack/v1/policies.py:103
#: senlin/tests/unit/apiv1/test_policies.py:409
msgid ""
"Updating the spec of a policy is not supported because it may cause state"
" conflicts in engine."
msgstr ""

#: senlin/api/openstack/v1/policies.py:110 senlin/engine/service.py:368
msgid "Policy name not specified."
msgstr ""

#: senlin/api/openstack/v1/profiles.py:34
msgid "No profile name specified"
msgstr ""

#: senlin/api/openstack/v1/profiles.py:39
msgid "No profile spec provided"
msgstr ""

#: senlin/api/openstack/v1/profiles.py:100
#: senlin/api/openstack/v1/profiles.py:123
msgid "Malformed request data, missing 'profile' key in request body."
msgstr ""

#: senlin/api/openstack/v1/receivers.py:38
msgid "Missing 'name' in request."
msgstr ""

#: senlin/api/openstack/v1/receivers.py:43
msgid "Missing 'cluster_id' in request."
msgstr ""

#: senlin/api/openstack/v1/receivers.py:48
msgid "Missing 'type' in request."
msgstr ""

#: senlin/api/openstack/v1/receivers.py:53
msgid "Missing 'action' in request."
msgstr ""

#: senlin/api/openstack/v1/receivers.py:114
msgid "Malformed request data, missing 'receiver' key in request body."
msgstr ""

#: senlin/cmd/manage.py:61
msgid "How long to preserve deleted data."
msgstr ""

#: senlin/cmd/manage.py:65
msgid "Unit to use for age argument, defaults to days."
msgstr ""

#: senlin/common/config.py:30
msgid "The API paste config file to use."
msgstr ""

#: senlin/common/config.py:35
msgid "Seconds between running periodic tasks."
msgstr ""

#: senlin/common/config.py:37
msgid "Default region name used to get services endpoints."
msgstr ""

#: senlin/common/config.py:40
msgid "Maximum raw byte size of data from web response."
msgstr ""

#: senlin/common/config.py:43
msgid "Maximum depth allowed when using nested clusters."
msgstr ""

#: senlin/common/config.py:46
msgid "Number of senlin-engine processes to fork and run."
msgstr ""

#: senlin/common/config.py:51
msgid "The directory to search for environment files."
msgstr ""

#: senlin/common/config.py:54
msgid "Maximum nodes allowed per top-level cluster."
msgstr ""

#: senlin/common/config.py:57
msgid "Maximum number of clusters any one project may have active at one time."
msgstr ""

#: senlin/common/config.py:61
msgid ""
"Maximum events per cluster. Older events will be deleted when this is "
"reached.  Set to 0 for unlimited events per cluster."
msgstr ""

#: senlin/common/config.py:66
msgid "Timeout in seconds for actions."
msgstr ""

#: senlin/common/config.py:69
msgid "Maximum number of actions per batch when operating a cluster."
msgstr ""

#: senlin/common/config.py:73
msgid "Number of times trying to grab a lock."
msgstr ""

#: senlin/common/config.py:76
msgid "Number of seconds between lock retries."
msgstr ""

#: senlin/common/config.py:79
msgid "Error wait time in seconds for cluster action (ie. create or update)."
msgstr ""

#: senlin/common/config.py:83
msgid ""
"RPC timeout for the engine liveness check that is used for cluster "
"locking."
msgstr ""

#: senlin/common/config.py:87
msgid ""
"Flag to indicate whether to enforce unique names for Senlin objects "
"belonging to the same project."
msgstr ""

#: senlin/common/config.py:93
msgid ""
"Name of the engine node. This can be an opaque identifier. It is not "
"necessarily a hostname, FQDN, or IP address."
msgstr ""

#: senlin/common/config.py:100
msgid "Default cloud backend to use."
msgstr ""

#: senlin/common/config.py:105
msgid "Complete public identity V3 API endpoint."
msgstr ""

#: senlin/common/config.py:107
msgid "Senlin service user name"
msgstr ""

#: senlin/common/config.py:109
msgid "Password specified for the Senlin service user."
msgstr ""

#: senlin/common/config.py:111
msgid "Name of the service project."
msgstr ""

#: senlin/common/config.py:113
msgid "Name of the domain for the service user."
msgstr ""

#: senlin/common/config.py:115
msgid "Name of the domain for the service project."
msgstr ""

#: senlin/common/config.py:121
msgid "Senlin API revision."
msgstr ""

#: senlin/common/config.py:123
msgid "Senlin engine revision."
msgstr ""

#: senlin/common/config.py:173
msgid "Unable to locate config file"
msgstr ""

#: senlin/common/config.py:185
#, python-format
msgid ""
"Unable to load %(app_name)s from configuration file %(conf_file)s.\n"
"Got: %(e)r"
msgstr ""

#: senlin/common/constraints.py:65
msgid "AllowedValues must be a list or a string"
msgstr ""

#: senlin/common/constraints.py:72
#, python-format
msgid "Allowed values: %s"
msgstr ""

#: senlin/common/constraints.py:76
#, python-format
msgid "\"%(value)s\" must be one of the allowed values: %(allowed)s"
msgstr ""

#: senlin/common/exception.py:41
msgid "An unknown exception occurred."
msgstr ""

#: senlin/common/exception.py:71
msgid "System SIGHUP signal received."
msgstr ""

#: senlin/common/exception.py:75
msgid "You are not authenticated."
msgstr ""

#: senlin/common/exception.py:79
msgid "You are not authorized to complete this action."
msgstr ""

#: senlin/common/exception.py:83
#: senlin/tests/unit/engine/service/test_clusters.py:704
#, python-format
msgid "The request is malformed: %(msg)s"
msgstr ""

#: senlin/common/exception.py:87
#, python-format
msgid ""
"Multiple results found matching the query criteria %(arg)s. Please be "
"more specific."
msgstr ""

#: senlin/common/exception.py:92
#, python-format
msgid "Invalid value '%(value)s' specified for '%(name)s'"
msgstr ""

#: senlin/common/exception.py:96
#, python-format
msgid "The cluster (%(cluster)s) could not be found."
msgstr ""

#: senlin/common/exception.py:100
#, python-format
msgid "The node (%(node)s) could not be found."
msgstr ""

#: senlin/common/exception.py:104
#, python-format
msgid "Profile type (%(profile_type)s) is not found."
msgstr ""

#: senlin/common/exception.py:108 senlin/common/exception.py:120
#: senlin/common/exception.py:124 senlin/common/exception.py:149
#: senlin/common/exception.py:153 senlin/common/exception.py:201
#: senlin/common/exception.py:256 senlin/common/exception.py:260
#, python-format
msgid "%(message)s"
msgstr ""

#: senlin/common/exception.py:112
#, python-format
msgid "The profile (%(profile)s) could not be found."
msgstr ""

#: senlin/common/exception.py:116
msgid "Profile not specified."
msgstr ""

#: senlin/common/exception.py:128 senlin/engine/actions/cluster_action.py:654
#: senlin/engine/actions/cluster_action.py:668
#: senlin/engine/actions/cluster_action.py:681
msgid "Policy not specified."
msgstr ""

#: senlin/common/exception.py:132
#, python-format
msgid "Policy type (%(policy_type)s) is not found."
msgstr ""

#: senlin/common/exception.py:136
#, python-format
msgid "The policy (%(policy)s) could not be found."
msgstr ""

#: senlin/common/exception.py:140
#, python-format
msgid ""
"The policy (%(policy)s) is not found attached to the specified cluster "
"(%(identity)s)."
msgstr ""

#: senlin/common/exception.py:145
#, python-format
msgid "The policy with type (%(policy_type)s) already exists."
msgstr ""

#: senlin/common/exception.py:157
#, python-format
msgid "%(feature)s is not supported."
msgstr ""

#: senlin/common/exception.py:168
#, python-format
msgid "The %(resource_type)s (%(resource_id)s) is still in use."
msgstr ""

#: senlin/common/exception.py:172
#, python-format
msgid "Invalid content type %(content_type)s"
msgstr ""

#: senlin/common/exception.py:176
#, python-format
msgid "Request limit exceeded: %(message)s"
msgstr ""

#: senlin/common/exception.py:180
#, python-format
msgid "The webhook (%(webhook)s) could not be found."
msgstr ""

#: senlin/common/exception.py:184
#, python-format
msgid "The receiver (%(receiver)s) could not be found."
msgstr ""

#: senlin/common/exception.py:188
#, python-format
msgid "The action (%(action)s) could not be found."
msgstr ""

#: senlin/common/exception.py:192
#, python-format
msgid "Cluster %(cluster_name)s already has an action (%(action)s) in progress."
msgstr ""

#: senlin/common/exception.py:197
#, python-format
msgid "The event (%(event)s) could not be found."
msgstr ""

#: senlin/common/exception.py:211
#, python-format
msgid "ERROR %(code)s happens for %(message)s."
msgstr ""

#: senlin/common/exception.py:212
msgid "Internal error happens"
msgstr ""

#: senlin/common/exception.py:222
#, python-format
msgid "The %(resource_type)s (%(resource_id)s) is busy now."
msgstr ""

#: senlin/common/exception.py:227
#, python-format
msgid "The trust for trustor (%(trustor)s) could not be found."
msgstr ""

#: senlin/common/exception.py:232
#, python-format
msgid "Failed in creating %(rtype)s."
msgstr ""

#: senlin/common/exception.py:237
#, python-format
msgid "Failed in updating %(resource)s."
msgstr ""

#: senlin/common/exception.py:242
#, python-format
msgid "Failed in deleting %(resource)s."
msgstr ""

#: senlin/common/exception.py:247
#, python-format
msgid "The resource (%(resource)s) could not be found."
msgstr ""

#: senlin/common/exception.py:251
#, python-format
msgid ""
"The resource %(resource_id)s is in error status - '%(status)s' due to "
"'%(reason)s'."
msgstr ""

#: senlin/common/exception.py:264
#, python-format
msgid ""
"The policy (%(policy)s) is not attached to the specified cluster "
"(%(cluster)s)."
msgstr ""

#: senlin/common/scaleutils.py:57
#, python-format
msgid "Truncating shrinkage to specified min_size (%s)."
msgstr ""

#: senlin/common/scaleutils.py:62
#, python-format
msgid "Truncating shrinkage to cluster's min_size (%s)."
msgstr ""

#: senlin/common/scaleutils.py:67
#, python-format
msgid "Truncating growth to specified max_size (%s)."
msgstr ""

#: senlin/common/scaleutils.py:73
#, python-format
msgid "Truncating growth to cluster's max_size (%s)."
msgstr ""

#: senlin/common/scaleutils.py:92
#, python-format
msgid "The target capacity (%(d)s) is less than the specified min_size (%(m)s)."
msgstr ""

#: senlin/common/scaleutils.py:97
#, python-format
msgid "The target capacity (%(d)s) is less than the cluster's min_size (%(m)s)."
msgstr ""

#: senlin/common/scaleutils.py:103
#, python-format
msgid ""
"The target capacity (%(d)s) is greater than the specified max_size "
"(%(m)s)."
msgstr ""

#: senlin/common/scaleutils.py:109
#, python-format
msgid ""
"The target capacity (%(d)s) is greater than the cluster's max_size "
"(%(m)s)."
msgstr ""

#: senlin/common/scaleutils.py:114
msgid "The specified min_size is greater than the specified max_size."
msgstr ""

#: senlin/common/scaleutils.py:119
msgid ""
"The specified min_size is greater than the current max_size of the "
"cluster."
msgstr ""

#: senlin/common/scaleutils.py:123
msgid ""
"The specified min_size is greater than the current desired_capacity of "
"the cluster."
msgstr ""

#: senlin/common/scaleutils.py:129
msgid "The specified max_size is less than the current min_size of the cluster."
msgstr ""

#: senlin/common/scaleutils.py:134
msgid ""
"The specified max_size is less than the current desired_capacity of the "
"cluster."
msgstr ""

#: senlin/common/schema.py:30
#, python-format
msgid "Invalid key %s"
msgstr ""

#: senlin/common/schema.py:64
#, python-format
msgid "Schema valid only for List or Map, not \"%s\""
msgstr ""

#: senlin/common/schema.py:95
#, python-format
msgid "Invalid default %(default)s (%(exc)s)"
msgstr ""

#: senlin/common/schema.py:171
#, python-format
msgid "The value \"%s\" is not a valid Boolean"
msgstr ""

#: senlin/common/schema.py:196
#, python-format
msgid "%s is not an intger."
msgstr ""

#: senlin/common/schema.py:204 senlin/tests/unit/test_common_constraints.py:283
#, python-format
msgid "The value \"%s\" cannot be converted into an integer."
msgstr ""

#: senlin/common/schema.py:233
#, python-format
msgid "The value \"%s\" cannot be converted into a string."
msgstr ""

#: senlin/common/schema.py:296 senlin/common/schema.py:302
#, python-format
msgid "\"%s\" is not a List"
msgstr ""

#: senlin/common/schema.py:310 senlin/common/schema.py:342
#: senlin/common/schema.py:348 senlin/common/schema.py:354
#, python-format
msgid "\"%s\" is not a Map"
msgstr ""

#: senlin/common/schema.py:374
#, python-format
msgid "Spec validation error (%(key)s): %(err)s"
msgstr ""

#: senlin/common/schema.py:380 senlin/tests/unit/test_common_constraints.py:296
#, python-format
msgid "Unrecognizable spec item \"%s\""
msgstr ""

#: senlin/common/schema.py:385
#, python-format
msgid "Invalid spec item: \"%s\""
msgstr ""

#: senlin/common/schema.py:394 senlin/tests/unit/test_common_constraints.py:309
#, python-format
msgid "Required spec item \"%s\" not assigned"
msgstr ""

#: senlin/common/schema.py:416
msgid "The provided spec is not a map."
msgstr ""

#: senlin/common/schema.py:420
msgid "The 'type' key is missing from the provided spec map."
msgstr ""

#: senlin/common/schema.py:424
msgid "The 'version' key is missing from the provided spec map."
msgstr ""

#: senlin/common/utils.py:116
#, python-format
msgid "Invalid URL scheme %s"
msgstr ""

#: senlin/common/utils.py:122 senlin/common/utils.py:145
#, python-format
msgid "Failed to retrieve data: %s"
msgstr ""

#: senlin/db/sqlalchemy/api.py:328
#, python-format
msgid "Node %(node)s: %(reason)s"
msgstr ""

#: senlin/db/sqlalchemy/api.py:907
msgid "Multiple dependencies between lists not support"
msgstr ""

#: senlin/db/sqlalchemy/api.py:919 senlin/db/sqlalchemy/api.py:938
msgid "Waiting for depended actions."
msgstr ""

#: senlin/db/sqlalchemy/api.py:951
msgid "Action completed successfully."
msgstr ""

#: senlin/db/sqlalchemy/api.py:969
msgid "All depended actions completed."
msgstr ""

#: senlin/db/sqlalchemy/api.py:983 senlin/db/sqlalchemy/api.py:1007
msgid "Action execution failed"
msgstr ""

#: senlin/db/sqlalchemy/api.py:1036
#, python-format
msgid "The action is not in an executable status: %s"
msgstr ""

#: senlin/db/sqlalchemy/api.py:1043 senlin/db/sqlalchemy/api.py:1060
msgid "The action is being processed."
msgstr ""

#: senlin/db/sqlalchemy/api.py:1078
msgid "The action was abandoned."
msgstr ""

#: senlin/drivers/openstack/lbaas.py:75
#, python-format
msgid "Waiting for loadbalancer %(lb)s to become ready"
msgstr ""

#: senlin/drivers/openstack/lbaas.py:229
msgid "LB deletion succeeded"
msgstr ""

#: senlin/engine/cluster.py:323
msgid "Policy already attached."
msgstr ""

#: senlin/engine/cluster.py:327
#, python-format
msgid ""
"Only one instance of policy type (%(ptype)s) can be attached to a "
"cluster, but another instance (%(existing)s) is found attached to the "
"cluster (%(cluster)s) already."
msgstr ""

#: senlin/engine/cluster.py:352
msgid "Policy attached."
msgstr ""

#: senlin/engine/cluster.py:371 senlin/engine/cluster.py:399
msgid "Policy not attached."
msgstr ""

#: senlin/engine/cluster.py:375
msgid "No update is needed."
msgstr ""

#: senlin/engine/cluster.py:380
msgid "Policy updated."
msgstr ""

#: senlin/engine/cluster.py:409
msgid "Policy detached."
msgstr ""

#: senlin/engine/environment.py:88
#, python-format
msgid "environment has unknown section \"%s\""
msgstr ""

#: senlin/engine/environment.py:107
#, python-format
msgid "%s type name not specified"
msgstr ""

#: senlin/engine/environment.py:110
#, python-format
msgid "%s type name is not a string"
msgstr ""

#: senlin/engine/environment.py:149
#, python-format
msgid "Driver plugin %(name)s is not found."
msgstr ""

#: senlin/engine/event.py:235
#, python-format
msgid "%(name)s [%(id)s] %(action)s - %(status)s: %(reason)s"
msgstr ""

#: senlin/engine/node.py:87
#, python-format
msgid "Profile not found: %s"
msgstr ""

#: senlin/engine/node.py:233
#, python-format
msgid "Profile failed in %(action)s resource (%(id)s) due to: %(msg)s"
msgstr ""

#: senlin/engine/node.py:238 senlin/tests/unit/engine/test_node.py:420
#, python-format
msgid "Profile failed in creating node due to: %(msg)s"
msgstr ""

#: senlin/engine/node.py:252
#, python-format
msgid "Failed in creating server: %s"
msgstr ""

#: senlin/engine/parser.py:73
#, python-format
msgid "Error parsing input: %s"
msgstr ""

#: senlin/engine/parser.py:80
msgid "The input is not a JSON object or YAML mapping."
msgstr ""

#: senlin/engine/service.py:199
#, python-format
msgid "The profile (%(name)s) already exists."
msgstr ""

#: senlin/engine/service.py:208
#, python-format
msgid "The specified profile type (%(name)s) is not supported."
msgstr ""

#: senlin/engine/service.py:320
#, python-format
msgid "The policy (%(name)s) already exists."
msgstr ""

#: senlin/engine/service.py:329
#, python-format
msgid "The specified policy type (%(name)s) is not supported."
msgstr ""

#: senlin/engine/service.py:445
msgid ""
"Cluster min_size, if specified, must be lesser than or equal to its "
"desired_capacity."
msgstr ""

#: senlin/engine/service.py:451
msgid ""
"Cluster max_size, if specified, must be greater than or equal to its "
"desired_capacity. Setting max_size to -1 means no upper limit on cluster "
"size."
msgstr ""

#: senlin/engine/service.py:471
#, python-format
msgid "The cluster (%(name)s) already exists."
msgstr ""

#: senlin/engine/service.py:478 senlin/engine/service.py:538
#, python-format
msgid "The specified profile '%s' is not found."
msgstr ""

#: senlin/engine/service.py:528
msgid "Updating a cluster in error state"
msgstr ""

#: senlin/engine/service.py:543
msgid "Cannot update a cluster to a different profile type, operation aborted."
msgstr ""

#: senlin/engine/service.py:621
#: senlin/tests/unit/engine/service/test_clusters.py:754
#, python-format
msgid "Profile type of nodes %s does not match with cluster"
msgstr ""

#: senlin/engine/service.py:626
#: senlin/tests/unit/engine/service/test_clusters.py:722
#: senlin/tests/unit/engine/service/test_clusters.py:731
#, python-format
msgid "Nodes %s already owned by some cluster"
msgstr ""

#: senlin/engine/service.py:630
#: senlin/tests/unit/engine/service/test_clusters.py:703
#, python-format
msgid "Nodes are not ACTIVE: %s"
msgstr ""

#: senlin/engine/service.py:632 senlin/engine/service.py:678
#, python-format
msgid "Nodes not found: %s"
msgstr ""

#: senlin/engine/service.py:634
#, python-format
msgid "No nodes to add: %s"
msgstr ""

#: senlin/engine/service.py:680
#, python-format
msgid "Nodes not members of specified cluster: %s"
msgstr ""

#: senlin/engine/service.py:682
msgid "No nodes specified."
msgstr ""

#: senlin/engine/service.py:747
msgid "Missing number value for size adjustment."
msgstr ""

#: senlin/engine/service.py:751
msgid "Missing adjustment_type value for size adjustment."
msgstr ""

#: senlin/engine/service.py:894
#, python-format
msgid "Cluster %(id)s cannot be deleted without having all policies detached."
msgstr ""

#: senlin/engine/service.py:957
#, python-format
msgid "The node (%(name)s) already exists."
msgstr ""

#: senlin/engine/service.py:969 senlin/engine/service.py:1046
#, python-format
msgid "The specified profile (%s) is not found."
msgstr ""

#: senlin/engine/service.py:976
#, python-format
msgid "The specified cluster (%s) is not found."
msgstr ""

#: senlin/engine/service.py:985
msgid "Node and cluster have different profile type, operation aborted."
msgstr ""

#: senlin/engine/service.py:1054
msgid "Cannot update a node to a different profile type, operation aborted."
msgstr ""

#: senlin/engine/service.py:1145 senlin/engine/service.py:1186
#: senlin/engine/service.py:1228
#, python-format
msgid "The specified policy (%s) is not found."
msgstr ""

#: senlin/engine/service.py:1192 senlin/engine/service.py:1234
#, python-format
msgid "The policy (%(p)s) is not attached to the specified cluster (%(c)s)."
msgstr ""

#: senlin/engine/service.py:1381
#, python-format
msgid "A receiver named '%s' already exists."
msgstr ""

#: senlin/engine/service.py:1390
#, python-format
msgid "Receiver type '%s' is not supported."
msgstr ""

#: senlin/engine/service.py:1398
#, python-format
msgid "The referenced cluster '%s' is not found."
msgstr ""

#: senlin/engine/service.py:1407
#, python-format
msgid "Illegal action '%s' specified."
msgstr ""

#: senlin/engine/service.py:1411
#, python-format
msgid "Action '%s' is not applicable to clusters."
msgstr ""

#: senlin/engine/service.py:1456
#, python-format
msgid "The referenced cluster (%s) is not found."
msgstr ""

#: senlin/engine/actions/base.py:258
#, python-format
msgid ""
"Action (%(action)s) is in unexpected status (%(actual)s) while expected "
"status should be one of (%(expected)s)."
msgstr ""

#: senlin/engine/actions/base.py:354
#, python-format
msgid "Failed policy '%(name)s': %(reason)s."
msgstr ""

#: senlin/engine/actions/base.py:387
msgid "Completed policy checking."
msgstr ""

#: senlin/engine/actions/base.py:408
#, python-format
msgid "Policy %s cooldown is still in progress."
msgstr ""

#: senlin/engine/actions/base.py:475
#, python-format
msgid ""
"Unexpected exception occurred during action %(action)s (%(id)s) "
"execution: %(reason)s"
msgstr ""

#: senlin/engine/actions/cluster_action.py:79
#, python-format
msgid "%(action)s [%(id)s] failed"
msgstr ""

#: senlin/engine/actions/cluster_action.py:87
#, python-format
msgid "%(action)s [%(id)s] cancelled"
msgstr ""

#: senlin/engine/actions/cluster_action.py:94
#, python-format
msgid "%(action)s [%(id)s] timeout"
msgstr ""

#: senlin/engine/actions/cluster_action.py:181
msgid "Cluster creation failed."
msgstr ""

#: senlin/engine/actions/cluster_action.py:188
msgid "Cluster creation succeeded."
msgstr ""

#: senlin/engine/actions/cluster_action.py:205
msgid "Cluster update failed."
msgstr ""

#: senlin/engine/actions/cluster_action.py:259
msgid "Cluster update completed."
msgstr ""

#: senlin/engine/actions/cluster_action.py:313
msgid "Deletion in progress."
msgstr ""

#: senlin/engine/actions/cluster_action.py:329
msgid "Cannot delete cluster object."
msgstr ""

#: senlin/engine/actions/cluster_action.py:354
#: senlin/engine/actions/cluster_action.py:431
#, python-format
msgid "Node [%s] is not found."
msgstr ""

#: senlin/engine/actions/cluster_action.py:357
#, python-format
msgid "Node [%(n)s] is already owned by cluster [%(c)s]."
msgstr ""

#: senlin/engine/actions/cluster_action.py:362
#, python-format
msgid "Node [%s] is not in ACTIVE status."
msgstr ""

#: senlin/engine/actions/cluster_action.py:370
msgid "Completed adding nodes."
msgstr ""

#: senlin/engine/actions/cluster_action.py:440
msgid "Completed deleting nodes."
msgstr ""

#: senlin/engine/actions/cluster_action.py:481
#, python-format
msgid "Cluster resizing failed: %s"
msgstr ""

#: senlin/engine/actions/cluster_action.py:510
msgid "Cluster resize succeeded."
msgstr ""

#: senlin/engine/actions/cluster_action.py:540
#, python-format
msgid "Invalid count (%s) for scaling out."
msgstr ""

#: senlin/engine/actions/cluster_action.py:541
#: senlin/engine/actions/cluster_action.py:554
#: senlin/engine/actions/cluster_action.py:595
#: senlin/engine/actions/cluster_action.py:613
#, python-format
msgid "Cluster scaling failed: %s"
msgstr ""

#: senlin/engine/actions/cluster_action.py:561
#: senlin/engine/actions/cluster_action.py:634
msgid "Cluster scaling succeeded."
msgstr ""

#: senlin/engine/actions/cluster_action.py:594
#, python-format
msgid "Invalid count (%s) for scaling in."
msgstr ""

#: senlin/engine/actions/cluster_action.py:604
#, python-format
msgid ""
"Triming count (%(count)s) to current cluster size (%(curr)s) for scaling "
"in"
msgstr ""

#: senlin/engine/actions/cluster_action.py:698
#: senlin/engine/actions/cluster_action.py:719
#, python-format
msgid "Policy check failure: %s"
msgstr ""

#: senlin/engine/actions/cluster_action.py:708
#, python-format
msgid "Unsupported action: %s."
msgstr ""

#: senlin/engine/actions/cluster_action.py:742
msgid "Failed in locking cluster."
msgstr ""

#: senlin/engine/actions/node_action.py:74
msgid "Node created successfully."
msgstr ""

#: senlin/engine/actions/node_action.py:76
msgid "Node creation failed."
msgstr ""

#: senlin/engine/actions/node_action.py:101
msgid "Node deleted successfully."
msgstr ""

#: senlin/engine/actions/node_action.py:103
msgid "Node deletion failed."
msgstr ""

#: senlin/engine/actions/node_action.py:113
msgid "Node updated successfully."
msgstr ""

#: senlin/engine/actions/node_action.py:115
msgid "Node update failed."
msgstr ""

#: senlin/engine/actions/node_action.py:138
msgid "Node successfully joined cluster."
msgstr ""

#: senlin/engine/actions/node_action.py:140
msgid "Node failed in joining cluster."
msgstr ""

#: senlin/engine/actions/node_action.py:163
msgid "Node successfully left cluster."
msgstr ""

#: senlin/engine/actions/node_action.py:165
msgid "Node failed in leaving cluster."
msgstr ""

#: senlin/engine/actions/node_action.py:175
#, python-format
msgid "Unsupported action: %s"
msgstr ""

#: senlin/engine/actions/node_action.py:197
msgid "Failed in locking cluster"
msgstr ""

#: senlin/engine/actions/node_action.py:215
msgid "Failed in locking node"
msgstr ""

#: senlin/policies/base.py:44
msgid "Name of the policy type."
msgstr ""

#: senlin/policies/base.py:48
msgid "Version number of the policy type."
msgstr ""

#: senlin/policies/base.py:52
msgid "A text description of policy."
msgstr ""

#: senlin/policies/base.py:56
msgid "Properties for the policy."
msgstr ""

#: senlin/policies/base.py:232
#, python-format
msgid "Policy not applicable on profile type: %s"
msgstr ""

#: senlin/policies/batch_policy.py:66
msgid "Minimum number of nodes in service when performing updates."
msgstr ""

#: senlin/policies/batch_policy.py:70
msgid "Maximum number of nodes that can be updated at the same time."
msgstr ""

#: senlin/policies/batch_policy.py:75
msgid "Number of seconds between update batches if any."
msgstr ""

#: senlin/policies/deletion_policy.py:85
msgid "Criteria used in selecting candidates for deletion"
msgstr ""

#: senlin/policies/deletion_policy.py:92
msgid ""
"Whether a node should be completely destroyed after deletion. Default to "
"True"
msgstr ""

#: senlin/policies/deletion_policy.py:97
msgid "Number of seconds before real deletion happens."
msgstr ""

#: senlin/policies/deletion_policy.py:101
msgid ""
"Whether the desired capacity of the cluster should be reduced along the "
"deletion. Default to False."
msgstr ""

#: senlin/policies/deletion_policy.py:189
msgid "Candidates generated"
msgstr ""

#: senlin/policies/health_policy.py:81
msgid "Policy aspect for node failure detection."
msgstr ""

#: senlin/policies/health_policy.py:84
msgid "Type of node failure detection."
msgstr ""

#: senlin/policies/health_policy.py:93
msgid ""
"Number of seconds between pollings. Only required when type is "
"'NODE_STATUS_POLLING'."
msgstr ""

#: senlin/policies/health_policy.py:103
msgid "Policy aspect for node failure recovery."
msgstr ""

#: senlin/policies/health_policy.py:106
msgid "List of actions to try for node recovery."
msgstr ""

#: senlin/policies/health_policy.py:108
msgid "Action to try for node recovery."
msgstr ""

#: senlin/policies/health_policy.py:115
msgid "List of services to be fenced."
msgstr ""

#: senlin/policies/health_policy.py:117
msgid "Service to be fenced."
msgstr ""

#: senlin/policies/lb_policy.py:99
msgid "LB pool properties."
msgstr ""

#: senlin/policies/lb_policy.py:102
msgid "Protocol used for load balancing."
msgstr ""

#: senlin/policies/lb_policy.py:109
msgid "Port on which servers are running on the nodes."
msgstr ""

#: senlin/policies/lb_policy.py:113
msgid "Name or ID of subnet for the port on which nodes can be connected."
msgstr ""

#: senlin/policies/lb_policy.py:118
msgid "Load balancing algorithm."
msgstr ""

#: senlin/policies/lb_policy.py:125
msgid "Administrative state of the pool."
msgstr ""

#: senlin/policies/lb_policy.py:129
msgid "Session pesistence configuration."
msgstr ""

#: senlin/policies/lb_policy.py:132
msgid "Type of session persistence implementation."
msgstr ""

#: senlin/policies/lb_policy.py:138
msgid "Name of cookie if type set to APP_COOKIE."
msgstr ""

#: senlin/policies/lb_policy.py:146
msgid "VIP address and port of the pool."
msgstr ""

#: senlin/policies/lb_policy.py:149
msgid "Name or ID of Subnet on which the VIP address will be allocated."
msgstr ""

#: senlin/policies/lb_policy.py:154
msgid "IP address of the VIP."
msgstr ""

#: senlin/policies/lb_policy.py:158
msgid "Maximum number of connections per second allowed for this VIP"
msgstr ""

#: senlin/policies/lb_policy.py:163
msgid "Protocol used for VIP."
msgstr ""

#: senlin/policies/lb_policy.py:170
msgid "TCP port to listen on."
msgstr ""

#: senlin/policies/lb_policy.py:174
msgid "Administrative state of the VIP."
msgstr ""

#: senlin/policies/lb_policy.py:247
msgid "LB resources deletion succeeded."
msgstr ""

#: senlin/policies/lb_policy.py:314
msgid "Failed in removing deleted node(s) from lb pool."
msgstr ""

#: senlin/policies/lb_policy.py:331
msgid "Failed in adding new node(s) into lb pool."
msgstr ""

#: senlin/policies/region_placement.py:87
msgid "List of regions to choose from."
msgstr ""

#: senlin/policies/region_placement.py:89
msgid "An region as a candidate."
msgstr ""

#: senlin/policies/region_placement.py:92
msgid "Name of a region."
msgstr ""

#: senlin/policies/region_placement.py:95
msgid "Weight of the region. The default is 100."
msgstr ""

#: senlin/policies/region_placement.py:99
msgid ""
"Maximum number of nodes in this region. The default is -1 which means no "
"cap set."
msgstr ""

#: senlin/policies/region_placement.py:234
msgid "No region is found usable."
msgstr ""

#: senlin/policies/region_placement.py:245
msgid "There is no feasible plan to accommodate all nodes."
msgstr ""

#: senlin/policies/scaling_policy.py:59
msgid ""
"Event that will trigger this policy. Must be one of CLUSTER_SCALE_IN and "
"CLUSTER_SCALE_OUT."
msgstr ""

#: senlin/policies/scaling_policy.py:67
msgid "Detailed specification for scaling adjustments."
msgstr ""

#: senlin/policies/scaling_policy.py:70
msgid "Type of adjustment when scaling is triggered."
msgstr ""

#: senlin/policies/scaling_policy.py:77
msgid "A number specifying the amount of adjustment."
msgstr ""

#: senlin/policies/scaling_policy.py:81
msgid ""
"When adjustment type is set to \"CHANGE_IN_PERCENTAGE\", this specifies "
"the cluster size will be decreased by at least this number of nodes."
msgstr ""

#: senlin/policies/scaling_policy.py:87
msgid ""
"Whether do best effort scaling when new size of cluster will break the "
"size limitation"
msgstr ""

#: senlin/policies/scaling_policy.py:127
msgid "Scaling request validated."
msgstr ""

#: senlin/policies/scaling_policy.py:140
#, python-format
msgid "Count (%(count)s) invalid for action %(action)s."
msgstr ""

#: senlin/policies/scaling_policy.py:149 senlin/policies/scaling_policy.py:158
#: senlin/tests/unit/policies/test_scaling_policy.py:274
msgid "Do best effort scaling."
msgstr ""

#: senlin/policies/scaling_policy.py:152
#: senlin/tests/unit/policies/test_scaling_policy.py:253
msgid "Attempted scaling below minimum size."
msgstr ""

#: senlin/policies/scaling_policy.py:161
msgid "Attempted scaling above maximum size."
msgstr ""

#: senlin/policies/vSphereDRS_policy.py:99
msgid "group where place nodes"
msgstr ""

#: senlin/policies/vSphereDRS_policy.py:102
msgid "the name of given server_group"
msgstr ""

#: senlin/policies/vSphereDRS_policy.py:105
msgid "the affinity or anti-affintiy control rule"
msgstr ""

#: senlin/policies/vSphereDRS_policy.py:188
msgid "Server group resources deletion succeeded"
msgstr ""

#: senlin/policies/zone_placement.py:87
msgid "List of availability zones to choose from."
msgstr ""

#: senlin/policies/zone_placement.py:89
msgid "An availability zone as candidate."
msgstr ""

#: senlin/policies/zone_placement.py:92
msgid "Name of an availability zone."
msgstr ""

#: senlin/policies/zone_placement.py:95
msgid "Weight of the availability zone (default is 100)."
msgstr ""

#: senlin/policies/zone_placement.py:220
msgid "No availability zone found available."
msgstr ""

#: senlin/profiles/base.py:42
msgid "Name of the profile type."
msgstr ""

#: senlin/profiles/base.py:46
msgid "Version number of the profile type."
msgstr ""

#: senlin/profiles/base.py:50
msgid "Properties for the profile."
msgstr ""

#: senlin/profiles/base.py:323
#, python-format
msgid "Failed at recovering obj: %s "
msgstr ""

#: senlin/profiles/os/heat/stack.py:45
msgid "A dictionary for specifying the customized context for stack operations"
msgstr ""

#: senlin/profiles/os/heat/stack.py:50
msgid "Heat stack template."
msgstr ""

#: senlin/profiles/os/heat/stack.py:54
msgid "Parameters to be passed to Heat for stack operations."
msgstr ""

#: senlin/profiles/os/heat/stack.py:58
msgid "Contents of files referenced by the template, if any."
msgstr ""

#: senlin/profiles/os/heat/stack.py:62
msgid ""
"A integer that specifies the number of minutes that a stack operation "
"times out."
msgstr ""

#: senlin/profiles/os/heat/stack.py:66
msgid "A boolean specifying whether a stack operation can be rolled back."
msgstr ""

#: senlin/profiles/os/heat/stack.py:71
msgid "A map that specifies the environment used for stack operations."
msgstr ""

#: senlin/profiles/os/heat/stack.py:107
#, python-format
msgid "Failed validate stack template due to \"%s\""
msgstr ""

#: senlin/profiles/os/nova/server.py:84
msgid "Customized security context for operating servers."
msgstr ""

#: senlin/profiles/os/nova/server.py:87
msgid "Password for the administrator account."
msgstr ""

#: senlin/profiles/os/nova/server.py:90
msgid "Whether the disk partition is done automatically."
msgstr ""

#: senlin/profiles/os/nova/server.py:94
msgid "Name of availability zone for running the server."
msgstr ""

#: senlin/profiles/os/nova/server.py:97 senlin/profiles/os/nova/server.py:113
msgid ""
"A list specifying the properties of block devices to be used for this "
"server."
msgstr ""

#: senlin/profiles/os/nova/server.py:100 senlin/profiles/os/nova/server.py:116
msgid ""
"A map specifying the properties of a block device to be used by the "
"server."
msgstr ""

#: senlin/profiles/os/nova/server.py:104
msgid "Block device name, should be <=255 chars."
msgstr ""

#: senlin/profiles/os/nova/server.py:107
msgid "Block device size in GB."
msgstr ""

#: senlin/profiles/os/nova/server.py:120
msgid "ID of the source image, snapshot or volume"
msgstr ""

#: senlin/profiles/os/nova/server.py:123
msgid "Volume source type, should be image, snapshot, volume or blank"
msgstr ""

#: senlin/profiles/os/nova/server.py:128
msgid "Volume destination type, should be volume or local"
msgstr ""

#: senlin/profiles/os/nova/server.py:133
msgid "Bus of the device."
msgstr ""

#: senlin/profiles/os/nova/server.py:136
msgid "Name of the device(e.g. vda, xda, ....)."
msgstr ""

#: senlin/profiles/os/nova/server.py:139
msgid "Size of the block device in MB(for swap) and in GB(for other formats)"
msgstr ""

#: senlin/profiles/os/nova/server.py:144
msgid "Specifies the disk file system format(e.g. swap, ephemeral, ...)."
msgstr ""

#: senlin/profiles/os/nova/server.py:148
msgid "Define the boot order of the device"
msgstr ""

#: senlin/profiles/os/nova/server.py:151
msgid "Type of the device(e.g. disk, cdrom, ...)."
msgstr ""

#: senlin/profiles/os/nova/server.py:154
msgid "Whether to delete the volume when the server stops."
msgstr ""

#: senlin/profiles/os/nova/server.py:161
msgid "Whether config drive should be enabled for the server."
msgstr ""

#: senlin/profiles/os/nova/server.py:164
msgid "ID of flavor used for the server."
msgstr ""

#: senlin/profiles/os/nova/server.py:171
msgid "ID of image to be used for the new server."
msgstr ""

#: senlin/profiles/os/nova/server.py:175
msgid "Name of Nova keypair to be injected to server."
msgstr ""

#: senlin/profiles/os/nova/server.py:178
msgid ""
"A collection of key/value pairs to be associated with the server created."
" Both key and value should be <=255 chars."
msgstr ""

#: senlin/profiles/os/nova/server.py:182
msgid "Name of the server."
msgstr ""

#: senlin/profiles/os/nova/server.py:186
msgid "List of networks for the server."
msgstr ""

#: senlin/profiles/os/nova/server.py:188
msgid "A map specifying the properties of a network for uses."
msgstr ""

#: senlin/profiles/os/nova/server.py:191
msgid "Name or ID of network to create a port on."
msgstr ""

#: senlin/profiles/os/nova/server.py:194
msgid "Port ID to be used by the network."
msgstr ""

#: senlin/profiles/os/nova/server.py:197
msgid "Fixed IP to be used by the network."
msgstr ""

#: senlin/profiles/os/nova/server.py:204
msgid "List of files to be injected into the server, where each."
msgstr ""

#: senlin/profiles/os/nova/server.py:206
msgid "A map specifying the path & contents for an injected file."
msgstr ""

#: senlin/profiles/os/nova/server.py:210
msgid "In-instance path for the file to be injected."
msgstr ""

#: senlin/profiles/os/nova/server.py:214
msgid "Contents of the file to be injected."
msgstr ""

#: senlin/profiles/os/nova/server.py:221
msgid ""
"A collection of key/value pairs to be associated with the Scheduler "
"hints. Both key and value should be <=255 chars."
msgstr ""

#: senlin/profiles/os/nova/server.py:226
msgid "List of security groups."
msgstr ""

#: senlin/profiles/os/nova/server.py:228
msgid "Name of a security group"
msgstr ""

#: senlin/profiles/os/nova/server.py:233
msgid "User data to be exposed by the metadata server."
msgstr ""

#: senlin/profiles/os/nova/server.py:392
#, python-format
msgid "Failed in updating server flavor: %s"
msgstr ""

#: senlin/profiles/os/nova/server.py:407
#, python-format
msgid "Failed in updating server image: %s"
msgstr ""

#: senlin/profiles/os/nova/server.py:427
#, python-format
msgid "Failed in updating server network: %s"
msgstr ""

#: senlin/profiles/os/nova/server.py:446
#, python-format
msgid "Failed in updating server metadata: %s"
msgstr ""

#: senlin/profiles/os/nova/server.py:458
#, python-format
msgid "Failed in updating server name: %s"
msgstr ""

#: senlin/profiles/os/nova/server.py:477
#, python-format
msgid "Server resizing failed, revert it: %s"
msgstr ""

#: senlin/profiles/os/nova/server.py:508
msgid "Updating Nova server with image set to None is not supported by Nova."
msgstr ""

#: senlin/profiles/os/nova/server.py:702
#, python-format
msgid "Failed at getting server: %s"
msgstr ""

#: senlin/profiles/os/nova/server.py:717
#, python-format
msgid "Failed at rebuilding server: %s"
msgstr ""

#: senlin/tests/functional/test_cluster_scaling.py:76
#: senlin/tests/functional/test_cluster_scaling.py:213
msgid "The target capacity (6) is greater than the cluster's max_size (5)."
msgstr ""

#: senlin/tests/functional/test_cluster_scaling.py:113
#: senlin/tests/functional/test_cluster_scaling.py:247
#: senlin/tests/unit/engine/actions/test_cluster_action.py:1754
msgid "The target capacity (0) is less than the cluster's min_size (1)."
msgstr ""

#: senlin/tests/functional/test_cluster_scaling.py:342
msgid "The target capacity (1) is less than the specified min_size (2)."
msgstr ""

#: senlin/tests/functional/test_cluster_scaling.py:380
msgid "The target capacity (4) is less than the specified min_size (5)."
msgstr ""

#: senlin/tests/functional/test_cluster_scaling.py:396
msgid "The target capacity (3) is greater than the specified max_size (0)."
msgstr ""

#: senlin/tests/functional/test_cluster_scaling.py:405
msgid "The specified min_size (5) is greater than the specified max_size (3)."
msgstr ""

#: senlin/tests/functional/test_scaling_policy.py:143
msgid "Policy check failure: Attempted scaling below minimum size."
msgstr ""

#: senlin/tests/functional/common/client.py:46
#, python-format
msgid ""
"%(msg)s\n"
"Status Code: %(status)s\n"
"Body: %(body)s"
msgstr ""

#: senlin/tests/functional/common/client.py:118
#, python-format
msgid "Doing authentication: auth_url %(auth_url)s, status %(http_status)s."
msgstr ""

#: senlin/tests/functional/common/client.py:123
#, python-format
msgid "Authentication failed: %s"
msgstr ""

#: senlin/tests/functional/common/client.py:149
#, python-format
msgid "Admin endpoint of clustering service in %(region)s was not found."
msgstr ""

#: senlin/tests/functional/common/client.py:162
#, python-format
msgid "request url %(url)s, status_code %(status)s, response body %(body)s"
msgstr ""

#: senlin/tests/functional/utils/senlin_test_profile.py:29
msgid "A dictionary for specifying the customized context"
msgstr ""

#: senlin/tests/functional/utils/senlin_test_profile.py:33
msgid "The first key of Senlin test profile schema"
msgstr ""

#: senlin/tests/functional/utils/senlin_test_profile.py:37
msgid "The second key of Senlin test profile schema"
msgstr ""

#: senlin/tests/unit/test_common_exception.py:23
#, python-format
msgid "Testing message %(text)s"
msgstr ""

#: senlin/tests/unit/test_common_scaleutils.py:180
msgid "The target capacity (11) is greater than the specified max_size (10)."
msgstr ""

#: senlin/tests/unit/apiv1/test_clusters.py:784
msgid "Invalid value '10min' specified for 'timeout'"
msgstr ""

#: senlin/tests/unit/apiv1/test_receivers.py:291
#, python-format
msgid "receiver obj_type (%s) is unsupported"
msgstr ""

#: senlin/tests/unit/apiv1/test_receivers.py:356
#, python-format
msgid "Illegal action (%s) specified."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:164
#, python-format
msgid "Failed in creating load balancer (%s)."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:177
msgid "Failed in getting subnet: GET FAILED."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:186
msgid "Failed in creating loadbalancer: CREATE FAILED."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:211
#, python-format
msgid "Failed in creating listener (%s)."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:229
msgid "Failed in creating lb listener: CREATE FAILED."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:256
#, python-format
msgid "Failed in creating pool (%s)."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:277
msgid "Failed in creating lb pool: CREATE FAILED."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:314
msgid "Failed in deleting healthmonitor: DELETE FAILED."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:332
msgid "Failed in deleting lb pool: DELETE FAILED."
msgstr ""

#: senlin/tests/unit/drivers/test_lbaas_driver.py:350
msgid "Failed in deleting listener: DELETE FAILED."
msgstr ""

#: senlin/tests/unit/engine/actions/test_cluster_action.py:1337
msgid "The target capacity (8) is less than the specified min_size (10)."
msgstr ""

#: senlin/tests/unit/engine/actions/test_cluster_action.py:1512
msgid "The target capacity (5) is greater than the cluster's max_size (4)."
msgstr ""

#: senlin/tests/unit/engine/service/test_clusters.py:95
msgid "The request is malformed: The cluster (c-1) already exists."
msgstr ""

#: senlin/tests/unit/engine/service/test_clusters.py:600
#, python-format
msgid ""
"The request is malformed: Cluster %(id)s cannot be deleted without having"
" all policies detached."
msgstr ""

#: senlin/tests/unit/engine/service/test_nodes.py:113
msgid "The request is malformed: The node (n-1) already exists."
msgstr ""

#: senlin/tests/unit/engine/service/test_policies.py:64
msgid "The request is malformed: The policy (p-1) already exists."
msgstr ""

#: senlin/tests/unit/engine/service/test_profiles.py:66
msgid "The request is malformed: The profile (p-1) already exists."
msgstr ""

#: senlin/tests/unit/engine/service/test_receivers.py:212
msgid "The request is malformed: A receiver named 'r1' already exists."
msgstr ""

#: senlin/tests/unit/engine/service/test_receivers.py:222
msgid "The request is malformed: Receiver type 'rocket' is not supported."
msgstr ""

#: senlin/tests/unit/policies/test_scaling_policy.py:233
msgid "Count (-2) invalid for action CLUSTER_SCALE_IN."
msgstr ""

#: senlin/tests/unit/policies/test_scaling_policy.py:321
msgid "Count (-1) invalid for action CLUSTER_SCALE_IN."
msgstr ""

#: senlin/tests/unit/profiles/test_nova_server.py:1001
msgid "Failed in updating FAKE_ID."
msgstr ""

